junit-jupiter-params/src/test/java/org/junit/jupiter/params/converter/DefaultArgumentConverterTests.java

```diff
@@ -99,6 +100,19 @@ void convertsStringsToPrimitiveTypes() {
 		assertConverts("42.2_3", double.class, 42.23);
 	}
 
+	@Test
+	void throwsExceptionOnInvalidStringForPrimitiveTypes() {
+		assertThatExceptionOfType(ArgumentConversionException.class).isThrownBy(
```

Comment:
Feel free to enforce custom formatting as mentioned in https://github.com/junit-team/junit5/pull/3178#discussion_r1118899666.

Otherwise, we'll likely do that after merging the PR.

Reply:
Ok done